lib/utils.js

```diff
@@ -375,5 +376,28 @@ const utils = (module.exports = {
     } else if (typeof process === 'object' && process.env) {
       return process.env[varName];
     }
+  },
+
+  createDeprecationWarning(message) {
+    let deprecationWarningDisplayed = false;
+
+    return () => {
+      if (deprecationWarningDisplayed) {
+        return;
+      }
+
+      deprecationWarningDisplayed = true;
+
+      let format = magicpen.defaultFormat;
+      if (format === 'html') {
+        // override given this will be output in the browser console
+        format = 'text';
+      }
+      console.warn(
+        magicpen()
+          .text(message, 'bgYellow', 'black')
+          .toString(format)
+      );
+    };
```

Comment:
You can get the output format this way:

```js
const format = expect.outputFormat()
```

So this should also work:

```js
const output = expect.output.clone(expect.outputFormat())
```

Reply:
@sunesimonsen that actually doesn't work currently - seems that the method is only allowed on the top-level expect.. should we change that?

Reply:
That would be okay with me. Just keep in mind that this method also allow you to give it an argument to change the default format.